From f454457e8f42bd2ceb6355d4a70bd5d19e4d4fa3 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Sun, 27 Dec 2020 09:42:54 -0700
Subject: [PATCH 03/11] webview: Hard no to persistent histograms

* It's probably some kind of bug somewhere that these are never
  deleted, but they don't really add value to us in the first
  place.
* Stop eating 4MB every single time any app that uses webview
  ever opens.
* These were guarded by a feature check before
  https://chromium.googlesource.com/chromium/src/+/b86102f2a243
---
 android_webview/browser/aw_field_trials.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/android_webview/browser/aw_field_trials.cc b/android_webview/browser/aw_field_trials.cc
index 7d3e926804d81..2dd8f2e9b6b8f 100644
--- a/android_webview/browser/aw_field_trials.cc
+++ b/android_webview/browser/aw_field_trials.cc
@@ -91,6 +91,7 @@ class AwFeatureOverrides {
 }  // namespace
 
 void AwFieldTrials::OnVariationsSetupComplete() {
+#if 0
   // Persistent histograms must be enabled ASAP, but depends on Features.
   base::FilePath metrics_dir;
   if (base::PathService::Get(base::DIR_ANDROID_APP_DATA, &metrics_dir)) {
@@ -98,6 +99,7 @@ void AwFieldTrials::OnVariationsSetupComplete() {
   } else {
     NOTREACHED_IN_MIGRATION();
   }
+#endif
 }
 
 // TODO(crbug.com/40271903): Consider to migrate all WebView feature overrides
-- 
2.46.1

